include/adm/serial/changed_ids.hpp

```diff
@@ -0,0 +1,183 @@
+#ifndef LIBADM_CHANGED_IDS_HPP
+#define LIBADM_CHANGED_IDS_HPP
+#include <adm/detail/auto_base.hpp>
+#include <adm/elements.hpp>
+
+namespace adm {
+  enum class Status { NEW, CHANGED, EXTENDED, EXPIRED };
```

Comment:
IMO `Status` is too generic to live directly in the `adm` namespace. Perhaps `IdStatus` or `ChangedIdStatus` would be better?